source/tutorials/building-smart-contract-only-dapps.md

```diff
@@ -0,0 +1,183 @@
+title: Building Smart Contract only DApps with Embark
+layout: tutorial
+---
+
+Building decentralized applications often involves many parts and components, such as Smart Contracts and a front-end, that have to play well together, in order to provide users the best experience possible. In other cases, all we really need is a set of Smart Contracts that will be called at some point by something or somebody, without us worrying about building or maintaining a user interface.
+
+Embark enables us to implement either of those scenarios and in this article we're going to explore how to build a decentralized applications where Smart Contracts are the primary focus.
+
+## Creating a Smart Contracts only application
+
+Before get started, let's make sure that Embark's command line tool is actually installed. Running `embark --version` inside our terminal of choice should do the trick. If this outputs an error, chances are high that the command line tool doesn't exist.
+
+To change that, all we have to do is using Node's package manager `npm`, using the following command:
+
+```
+$ npm install -g embark@next
+```
+
+This will make Embark's command line tool globally available on our machines. Notice that `@next` picks the latest development version of Embark. This release tag will be removed in the future once version `4.0.0` stable has been released (no worries, we'll update this article accordingly). 
+
+If we don't feel comfortable with that, we can omit the `@next` part and get the latest stable version instead, which at the time of writing this article was `3.2.7`. However, we do recommend using the development version as it's fairly stable and comes with tons of improvements, better performance and features.
```

Comment:
Yes, similar to what @alaibe mentioned. I was thinking back and forth whether the article should be truly "timeless" at the cost of having people use v3.2.7 instead of v4.

As commented here https://github.com/embark-framework/embark-site/pull/130#discussion_r249360770, we'll have to go through the docs anyways to remove all `@next`'s once v4 is out, so having that in mind... would you be okay with leaving this in assuming that I'll take care of removing it once v4 stable is out?

I really believe we should have people give v4 a spin in it's current state.